src/main/charts/crowd/values.yaml

```diff
@@ -213,6 +213,22 @@ crowd:
     #
     usePodNameAsClusterNodeName: true
 
+  shutdown:
+
+    # -- The termination grace period for pods during shutdown. This
+    # should be set to the internal grace period, plus a small buffer
+    # to allow the JVM to fully terminate.
```

Comment:
nit - for consistency can we include a `#` after the the property name for all of these i.e.

```
# -- The termination grace period for pods during shutdown. This
# should be set to the internal grace period, plus a small buffer
# to allow the JVM to fully terminate.
#
terminationGracePeriodSeconds: 30
```

Reply:
Done